src/hotspot/share/prims/stackwalk.cpp

```diff
@@ -498,7 +498,7 @@ oop StackWalk::fetchFirstBatch(BaseFrameStream& stream, Handle stackStream,
   if (!stream.at_end()) {
     KeepStackGCProcessedMark keep_stack(THREAD);
     numFrames = fill_in_frames(mode, stream, frame_count, start_index,
-                               frames_array, end_index, CHECK_NULL);
+                               frames_array, end_index, true, CHECK_NULL);
```

Comment:
Could you annotate the new argument please ie. `true /* first batch */` and `false /* not first batch */`. Thanks.

Reply:
I've removed these arguments in the latest version.